pkg/collector/corechecks/containers/containerd/check.go

```diff
@@ -95,6 +95,10 @@ func (c *ContainerdCheck) Configure(senderManager sender.SenderManager, integrat
 		return err
 	}
 
+	if c.instance.OpenmetricsEndpoint == "" {
+		log.Warnf("openmetrics_endpoint configuration parameter is missing. Some metrics can not be generated in the absence of this configuration parameter.")
```

Comment:
We cannot send a warning because of unused feature. Especially as it requires `hostNetwork` which most customers are not willing to give to the Agent.

Reply:
Alright. 

I will remove the warning message and update the PR.

Thank you.